app/controllers/answers_controller.rb

```diff
@@ -1,4 +1,6 @@
 class AnswersController < ApplicationController
+
+  before_action :authenticate_user!
```

Comment:
Usually this line adds to the application_controller or base_controller to avoid paste it in all controllers, and just add skip_before_action to public page

Reply:
Ok, it will be fixed in the next pull request